Bottom: 0222a92c911b9a7e0cdcc9f5021665928a669803
Top:    9b85afcf26d3db7711c4d9254af03f2727a7e4b8
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-23 17:00:36 +0100

Refresh of draw-c-change-drawtop-to-drawo

---

diff --git a/src/draw.c b/src/draw.c
index f7a5c7c..f67aa29 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -414,9 +414,11 @@ DrawEverything (BoxTypePtr drawn_area)
   if (TEST_FLAG (CHECKPLANESFLAG, PCB) && gui->gui)
     return;
 
+  side = SWAP_IDENT ? SOLDER_LAYER : COMPONENT_LAYER;
+
   /* Draw pins, pads, vias below silk */
   if (gui->gui)
-    DrawTop (drawn_area);
+    DrawOuterLayer (side, drawn_area);
   else
     {
       HoleCountStruct hcs;
@@ -581,10 +583,8 @@ pad_callback (const BoxType * b, void *cl)
  * draws pins pads and vias
  */
 static void
-DrawTop (const BoxType * screen)
+DrawOuterLayer (side, const BoxType * screen)
 {
-  int side = SWAP_IDENT ? SOLDER_LAYER : COMPONENT_LAYER;
-
   if (PCB->PinOn || doing_assy)
     {
       /* draw element pins */
